source/fundamentals/run-command.txt

```diff
@@ -55,17 +55,46 @@ the current member's role in the replica set, on a database:
 For a full list of database commands and corresponding parameters, see
 the :ref:`Additional Information section <addl-info-runcommand>`.
 
+.. _node-command-options:
+
+Command Options
+---------------
+
+You can set options when executing a database command by
+passing an options object to either command method. The ``command()`` method
+takes a ``RunCommandOptions`` object, and the ``runCursorCommand()`` method
+takes a ``RunCursorCommandOptions`` object.
+
+Starting in version 6.0 of the {+driver-short+}, you can pass only the
+following options directly to the ``command()`` method:
```

Comment:
Question: Does this apply to the runCursorCommand() method or only command() method? In either case, I think it could be good to call out the limitations for both methods.

Reply:
the ticket only says that it is for the `command()` method and from the feature PR, it seems  that `runCursorCommand()` is unchanged.